pkg/controller/testutil/test_utils.go

```diff
@@ -422,9 +422,10 @@ func (f *FakeRecorder) AnnotatedEventf(obj runtime.Object, annotations map[strin
 func (f *FakeRecorder) generateEvent(obj runtime.Object, timestamp metav1.Time, eventtype, reason, message string) {
 	f.Lock()
 	defer f.Unlock()
+	ctx := context.TODO()
```

Comment:
Resolving this will depend on API changes for event recorder. TODO is fine for now.